test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+      // only one object
+      assertIsNull((Object) null);
+      assertNotNull(1);
+      assertNotNull("abc");
+
+      // some multiple objects
+      assertNotNull("abc", 1, "ABC");
+      assertIsNull("abc", 1, null);
+      assertIsNull("abc", null, null);
+      assertIsNull(null, null, null);
```

Comment:
Noticed this in passing. You should be testing the SUT `Utils.isAnyNull` which exists already; just need to write the tests.